contrib/adsk/libraries/adsklib/adsklib_defs.mtlx

```diff
@@ -161,27 +161,27 @@
     <input name="surface_normal" type="vector3" value="1, 1, 1" defaultgeomprop="Nworld"/>
     <input name="surface_cutout" type="color3" value="1, 1, 1" />
     <input name="surface_albedo" type="color3" value="1, 1, 1" />
-    <input name="surface_roughness" type="float" value="0" />
-    <input name="surface_anisotropy" type="float" value="0" />
-    <input name="surface_rotation" type="float" value="0" />
-    <input name="transparent_ior" type="float" value="1" />
+    <input name="surface_roughness" type="float" value="0" uimin="0.0" uimax="1.0" />
+    <input name="surface_anisotropy" type="float" value="0" uimin="0.0" uimax="1.0" />
+    <input name="surface_rotation" type="float" value="0" uimin="0.0" uimax="180.0" />
+    <input name="transparent_ior" type="float" value="1" uimin="0.0" uisoftmax="3.0" />
```

Comment:
consult with Rob Z. on a better IOR range for the library presets. Perhaps we can go beyond 3

Reply:
Initially I put 10 as the limit. I then reduced to 3 so the initial range is closer to common values. Note it's a softmax, so you can go beyond 3.